hybrasyl/Time.cs

```diff
@@ -98,8 +98,8 @@ public long HybrasylTicks
 
         public long TerranTicks => HybrasylTicks / 8;
 
-        public static string DefaultAge => Game.Config.Time != null ? (Game.Config.Time.ServerStart.DefaultAge != string.Empty ? Game.Config.Time.ServerStart.DefaultAge : "Hybrasyl") : "Hybrasyl";
-        public static int DefaultYear => Game.Config.Time.ServerStart.DefaultYear != 1 ? Game.Config.Time.ServerStart.DefaultYear : 1;
+        public static string DefaultAge => Game.Config != null ? Game.Config.Time.ServerStart.DefaultAge != string.Empty ? Game.Config.Time.ServerStart.DefaultAge : "Hybrasyl" : "Hybrasyl";
+        public static int DefaultYear => Game.Config != null ? Game.Config.Time.ServerStart.DefaultYear != 1 ? Game.Config.Time.ServerStart.DefaultYear : 1 : 1;
```

Comment:
this should be fixed later, Config should never be null and you can use coalesce here